Exclude the spring-boot-starter-logging dependency from the azure adapter

**Describe the bug**
Apparently the `org.springframework.boot:spring-boot-starter-logging` dependency inherited by the `spring-boot-starter` leads to conflicts with the Azure function rutime

**Sample**
With the spring-boot-starter-logging present one may observe the following errors when running the azure function:
```
For detailed output, run func with --verbose flag.
[2022-10-21T13:12:28.880Z] Worker process started and initialized.
[2022-10-21T13:12:28.928Z] SLF4J: No SLF4J providers were found.
[2022-10-21T13:12:28.928Z] SLF4J: Defaulting to no-operation (NOP) logger implementation
[2022-10-21T13:12:28.929Z] SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
[2022-10-21T13:13:10.420Z] Executing 'Functions.KafkaTrigger' (Reason='(null)', Id=bb5af91d-2004-4c62-b8e8-ed610b29eb24)
[2022-10-21T13:13:10.637Z] Executed 'Functions.KafkaTrigger' (Failed, Id=bb5af91d-2004-4c62-b8e8-ed610b29eb24, Duration=253ms)
[2022-10-21T13:13:10.637Z] System.Private.CoreLib: Exception while executing function: Functions.KafkaTrigger. System.Private.CoreLib: Result: Failure
[2022-10-21T13:13:10.637Z] Exception: IllegalArgumentException: LoggerFactory is not a Logback LoggerContext but Logback is on the classpath. Either remove Logback or the competing implementation (class org.slf4j.helpers.NOPLoggerFactory loaded from file:/Users/ctzolov/Dev/projects/spring-cloud/spring-cloud-function/spring-cloud-function-samples/function-sample-azure-kafka-trigger/target/azure-functions/kafka-trigger-azure-spring-function/lib/slf4j-api-2.0.3.jar). If you are using WebLogic you will need to add 'org.slf4j' to prefer-application-packages in WEB-INF/weblogic.xml: org.slf4j.helpers.NOPLoggerFactory
[2022-10-21T13:13:10.637Z] Stack: java.lang.IllegalStateException: Failed to initialize
[2022-10-21T13:13:10.637Z]      at org.springframework.cloud.function.adapter.azure.FunctionInvoker.<init>(FunctionInvoker.java:88)
[2022-10-21T13:13:10.637Z]      at org.springframework.cloud.function.adapter.azure.FunctionInvoker.<init>(FunctionInvoker.java:93)
[2022-10-21T13:13:10.637Z]      at example.UppercaseHandler.<init>(UppercaseHandler.java:31)
[2022-10-21T13:13:10.637Z]      at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2022-10-21T13:13:10.637Z]      at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2022-10-21T13:13:10.637Z]      at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2022-10-21T13:13:10.637Z]      at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[2022-10-21T13:13:10.637Z]      at java.base/java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:128)
[2022-10-21T13:13:10.637Z]      at java.base/jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:347)
[2022-10-21T13:13:10.637Z]      at java.base/java.lang.Class.newInstance(Class.java:645)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.broker.EnhancedJavaMethodExecutorImpl.lambda$execute$1(EnhancedJavaMethodExecutorImpl.java:55)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.broker.JavaMethodInvokeInfo.invoke(JavaMethodInvokeInfo.java:20)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.broker.EnhancedJavaMethodExecutorImpl.execute(EnhancedJavaMethodExecutorImpl.java:55)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.broker.JavaFunctionBroker.invokeMethod(JavaFunctionBroker.java:61)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.handler.InvocationRequestHandler.execute(InvocationRequestHandler.java:33)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.handler.InvocationRequestHandler.execute(InvocationRequestHandler.java:10)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.handler.MessageHandler.handle(MessageHandler.java:45)
[2022-10-21T13:13:10.637Z]      at com.microsoft.azure.functions.worker.JavaWorkerClient$StreamingMessagePeer.lambda$onNext$0(JavaWorkerClient.java:92)
[2022-10-21T13:13:10.637Z]      at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2022-10-21T13:13:10.637Z]      at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2022-10-21T13:13:10.637Z]      at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2022-10-21T13:13:10.637Z]      at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2022-10-21T13:13:10.637Z]      at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-21T13:13:10.637Z] Caused by: java.lang.IllegalArgumentException: LoggerFactory is not a Logback LoggerContext but Logback is on the classpath. Either remove Logback or the competing implementation (class org.slf4j.helpers.NOPLoggerFactory loaded from file:/Users/ctzolov/Dev/projects/spring-cloud/spring-cloud-function/spring-cloud-function-samples/function-sample-azure-kafka-trigger/target/azure-functions/kafka-trigger-azure-spring-function/lib/slf4j-api-2.0.3.jar). If you are using WebLogic you will need to add 'org.slf4j' to prefer-application-packages in WEB-INF/weblogic.xml: org.slf4j.helpers.NOPLoggerFactory
[2022-10-21T13:13:10.637Z]      at org.springframework.util.Assert.instanceCheckFailed(Assert.java:702)
[2022-10-21T13:13:10.638Z]      at org.springframework.util.Assert.isInstanceOf(Assert.java:621)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.logging.logback.LogbackLoggingSystem.getLoggerContext(LogbackLoggingSystem.java:369)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.logging.logback.LogbackLoggingSystem.beforeInitialize(LogbackLoggingSystem.java:121)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationStartingEvent(LoggingApplicationListener.java:238)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:220)
[2022-10-21T13:13:10.638Z]      at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176)
[2022-10-21T13:13:10.638Z]      at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169)
[2022-10-21T13:13:10.638Z]      at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143)
[2022-10-21T13:13:10.638Z]      at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:131)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.context.event.EventPublishingRunListener.multicastInitialEvent(EventPublishingRunListener.java:136)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.context.event.EventPublishingRunListener.starting(EventPublishingRunListener.java:76)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.SpringApplicationRunListeners.lambda$starting$0(SpringApplicationRunListeners.java:54)
[2022-10-21T13:13:10.638Z]      at java.base/java.lang.Iterable.forEach(Iterable.java:75)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.SpringApplicationRunListeners.doWithListeners(SpringApplicationRunListeners.java:118)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.SpringApplicationRunListeners.starting(SpringApplicationRunListeners.java:54)
[2022-10-21T13:13:10.638Z]      at org.springframework.boot.SpringApplication.run(SpringApplication.java:300)
[2022-10-21T13:13:10.638Z]      at org.springframework.cloud.function.adapter.azure.FunctionInvoker.initialize(FunctionInvoker.java:349)
[2022-10-21T13:13:10.638Z]      at org.springframework.cloud.function.adapter.azure.FunctionInvoker.<init>(FunctionInvoker.java:84)
[2022-10-21T13:13:10.638Z]      ... 22 more
[2022-10-21T13:13:10.638Z] .
```

--- a/spring-cloud-function-adapters/spring-cloud-function-adapter-azure/pom.xml
+++ b/spring-cloud-function-adapters/spring-cloud-function-adapter-azure/pom.xml
@@ -31,6 +31,12 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter</artifactId>
+			<exclusions>
+				<exclusion>
+					<artifactId>spring-boot-starter-logging</artifactId>
+					<groupId>org.springframework.boot</groupId>
+				</exclusion>
+			</exclusions>
 		</dependency>
 		<dependency>
 			<groupId>io.projectreactor</groupId>

--- a/spring-cloud-function-samples/function-sample-azure-kafka-trigger/pom.xml
+++ b/spring-cloud-function-samples/function-sample-azure-kafka-trigger/pom.xml
@@ -36,12 +36,6 @@
 		<dependency>
 			<artifactId>spring-cloud-function-adapter-azure</artifactId>
 			<groupId>org.springframework.cloud</groupId>
-			<exclusions>
-				<exclusion>
-					<groupId>org.springframework.boot</groupId>
-					<artifactId>spring-boot-starter-logging</artifactId>
-				</exclusion>
-			</exclusions> 
 		</dependency>
 
 		<dependency>

--- a/spring-cloud-function-samples/function-sample-azure-timer-trigger/pom.xml
+++ b/spring-cloud-function-samples/function-sample-azure-timer-trigger/pom.xml
@@ -36,12 +36,6 @@
 		<dependency>
 			<artifactId>spring-cloud-function-adapter-azure</artifactId>
 			<groupId>org.springframework.cloud</groupId>
-			<exclusions>
-				<exclusion>
-					<groupId>org.springframework.boot</groupId>
-					<artifactId>spring-boot-starter-logging</artifactId>
-				</exclusion>
-			</exclusions> 
 		</dependency>
 
 		<dependency>

